install/threesdk/lib/SDKContainers.py

```diff
@@ -16,6 +17,9 @@ def __init__(self, core, args):
         self._wireguard = None
 
     def _get_user(self):
+        response = requests.get(f"https://login.threefold.me/api/users/{self.args.identity}")
+        if response.status_code == 404:
+            return "not_in_3botApp"
```

Comment:
we can and the end of this method already compare if the user his public keys match on both systems
if they do not match you can raise Tools.exceptions.Value(... explaining his keys do not match) nothing can be done at this point except go cry to support